### ISSUE TITLE
Personal Cabinet

### ISSUE THREAD

**Issue by acidmaksim** [Hexlet/hexlet-editor#73]
You can use https://replit.com/~ as a basis.

**Comment by Syrupred** [comment]
беру в работу

**Comment by acidmaksim** [comment]
@Syrupred как успехи с задаче?

**Comment by Tsogoeva** [comment]
Беру в работу

**Comment by Tsogoeva** [comment]
Правильно ли я понимаю, сделать отдельную страницу с профилем пользователя?
На подобие:
<img width="1434" alt="Screenshot 6:09:22" src="https://user-images.githubusercontent.com/52177557/188663808-8752a456-38d4-4494-9e2b-1ea11e4593ef.png">

**Comment by acidmaksim** [comment]
Да, за пример можно взять  replit. Но у нас еще есть задачки более приоритетные)

**Comment by acidmaksim** [comment]
@Tsogoeva там вроде роуты завезли в проект. Делаешь задачку?

**Comment by Tsogoeva** [comment]
Да, займусь этим

**Comment by Tsogoeva** [comment]
@acidmaksim хочу уточнить текущую задачу. Поскольку завезли роуты, то сейчас нужно добавить в navbar переход в кабинет пользователя?
Если нет, то подскажи, что требуется сделать на этом этапе.

**Comment by acidmaksim** [comment]
@Tsogoeva по сути да, потому что сейчас там нет ничего

**Comment by acidmaksim** [comment]
@Tsogoeva получается?

**Comment by Tsogoeva** [comment]
@acidmaksim я пока сделала перерыв по работе над проектом
Эта задача свободна

**Comment by Takaiva** [comment]
Занимаюсь!
